hw25/main.js

```diff
@@ -0,0 +1,56 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const resTasks = [
+        document.getElementById("res1"),
+        document.getElementById("res2"),
+        document.getElementById("res3"),
+        document.getElementById("res4"),
+        document.getElementById("res5"),
+        document.getElementById("res6"),
+    ];
+
+    //Task 1
+    {
+        function countVowels(inputString) {
+            const vowels = "aeiouyAEIOUY";
+            return `Vowels: ${inputString.split('').filter(char => vowels.includes(char)).length}`
+        };
```

Comment:
отлично, можно убрать заглавные буквы ( но это не обязательно, твой вариант отличный)
```suggestion
        function countVowels(inputString) {
            const vowels = "aeiouy";
            return `Vowels: ${inputString.toLowerCase().split('').filter(char => vowels.includes(char)).length}`
        };
```